fuel/schemes.py

```diff
@@ -229,6 +229,57 @@ def get_request_iterator(self):
             return imap(list, partition_all(self.batch_size, indices))
 
 
+class BalancedSamplingScheme(ShuffledScheme):
+    """Balanced sampling batches iterator.
+
+    Samples an equal amount of examples from each group and iterates over them
+    in shuffled batches.
+
+    Parameters
+    ----------
+    targets : list of int
+        The targets that represents the groups membership per class.
+    samples_per_class : int, optional
+        If `True`, enforce that indices within a batch are ordered.
+        Defaults to `False`.
```

Comment:
This is no longer a flag right? What happens when samples_per_class is not given?